examples/AspNetCore/README.md

```diff
@@ -0,0 +1,4 @@
+# OpenTelemetry ASP.NET Core Example Application
```

Comment:
probably don't need this!

Reply:
It was used to add a meta link to this folder. But I've removed it after removing that link.